### PR TITLE
DOC-10053: Release notes for 7.1.1

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -1,6 +1,100 @@
...
+This release contains the fixes listed below.
+
+=== Data Service
+
+[#table-known-issues-711-dataservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52248[MB-52248^]
+| Memcached hangs when no password is passed for encrypted private key
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc]: The terminology used in the docs etc for encrypted private keys, is that they are unlocked with a passphrase not a password. Ref: https://docs.couchbase.com/server/current/rest-api/upload-retrieve-node-cert.html#json-passphrase-registration

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -1,6 +1,100 @@
...
+|===
+
+=== Analytics Service
+
+[#table-known-issues-711-analyticsservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52008[MB-52008^]
+| Analytics average ingestion rate with magma is 90% lower compared to Couchstore
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc]: Should remove this item.  Analytics wasn't certified to use magma in previous releases, so this isn't something that customers need to be aware of.

File: modules/release-notes/pages/relnotes.adoc [hunk 3]
```diff
@@ -1,6 +1,100 @@
...
+|===
+
+=== XDCR
+
+[#table-known-issues-711-xdcr, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51939[MB-51939^]
+| XDCR does not update memcached flag/body after txn xattribute removal if user xattr DNE
```
Comment (nelio2k) [inline modules/release-notes/pages/relnotes.adoc]: For outward-facing documentation, let's replace DNE with "is not found"

File: modules/release-notes/pages/relnotes.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: I've changed.

File: modules/release-notes/pages/relnotes.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: I've removed.

File: modules/release-notes/pages/relnotes.adoc (hunk 3 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: I've changed.

**Pull request merged**